README.md

```diff
@@ -61,6 +61,11 @@ cmake --build . --config Release
 ### Building for macOS & iOS ###
 If you don't have CMake installed on your system, you can install it using brew with the following command: `brew install cmake`.
 
+You will also need the icu4c library installed.
```

Comment:
It's not required to have ICU installed. I want to keep it optional because ICU is so huge and it wouldn't make sense to add all that weight to the executable just to be able to decode Shift-JIS strings. Most OS have good built-in functions that can be used. Linux/FreeBSD doesn't provide anything built-in and that's where ICU is handy.